docs/components/sidecar.md

```diff
@@ -79,6 +79,9 @@ To use this, the Prometheus compaction needs to be disabled. This can be done by
 - `--storage.tsdb.min-block-duration=2h`
 - `--storage.tsdb.max-block-duration=2h`
 
+A known issue is that Prometheus and Thanos use different compaction schedules, so Compact may
```

Comment:
This is nice, can you link issue as well?

However, for documentation would be nice to be clear what we mean by "confused".. until we know roughly what can go wrong this piece of documentation will be only source of further questions.

Reply:
A agree the language is vague, but it's because we actually don't know.  See my other comment about not knowing how the compaction library deals with blocks mixed together from different compaction schedules.

I guess that's why `upload_compacted` is considered experimental...

Reply:
Well then we should either learn or not describe until we learn.

We have some idea though as you mentioned below